build.gradle.kts

```diff
@@ -74,7 +74,25 @@ apply(from = "gradle/verifier.gradle")
 
 extra["skipPublish"] = mutableListOf(
     "ktor-server-test-base",
-    "ktor-junit"
+    "ktor-server-test-suites",
+    "ktor-server-tests",
+    "ktor-junit",
+)
+extra["relocatedArtifacts"] = mutableMapOf(
+    "ktor-server-auth" to "ktor-auth",
```

Comment:
**Nit:** `"ktor-server-auth" to "ktor-auth"` is a bit confusing. It looks like `ktor-server-auth` was relocated to `ktor-auth`. But honestly, I don't have any proposition here. Maybe another name for the extra could make it better, but I can't come up with anything better than `"newToOldArtifactMapping"` which looks cumbersome

Reply:
I was thinking the same, actually.  I'll replace it with `mapOf(...).invert()` so that the declaration is more readable and the index is still correct.